diff --git a/types/mirage_types_lwt.ml b/types/mirage_types_lwt.ml
index 5345a21..db2f512 100644
--- a/types/mirage_types_lwt.ml
+++ b/types/mirage_types_lwt.ml
@@ -31,7 +31,7 @@ module type FLOW = Mirage_flow_lwt.S
 module type CONSOLE = Mirage_console_lwt.S
 
 (** Block devices *)
-module type BLOCK = Mirage_block_lwt.S
+(* module type BLOCK = Mirage_block_lwt.S *)
 
 (** Network *)
 module type NETWORK = Mirage_net_lwt.S
@@ -55,8 +55,8 @@ module type TCPV6 = Mirage_protocols_lwt.TCPV6
 module type CHANNEL = Mirage_channel_lwt.S
 
 (** Key-value stores *)
-module type KV_RO = Mirage_kv_lwt.RO
-module type KV_RW = Mirage_kv_lwt.RW
+(* module type KV_RO = Mirage_kv_lwt.RO
+module type KV_RW = Mirage_kv_lwt.RW *)
 
 (** FS *)
 module type FS = Mirage_fs_lwt.S
